apps/nowcasting-app/pages/index.tsx

```diff
@@ -164,29 +186,188 @@ export default function Home({ dashboardModeServer }: { dashboardModeServer: str
     isValidating: allGspSystemValidating,
     error: allGspSystemError
   } = useLoadDataFromApi<components["schemas"]["Location"][]>(`${API_PREFIX}/system/GB/gsp/`);
+
+  //////////////////////////////////////
+  // ALL GSP FORECAST DATA
+  //////////////////////////////////////
+  // Load future all GSP forecast data (every 5 minutes)
   const {
-    data: allGspForecastData,
-    isLoading: allGspForecastLoading,
-    isValidating: allGspForecastValidating,
-    error: allGspForecastError
+    data: allGspForecastFutureData,
+    isLoading: allGspForecastFutureLoading,
+    isValidating: allGspForecastFutureValidating,
+    error: allGspForecastFutureError
   } = useLoadDataFromApi<
     // TODO: see if we can fully integrate API paths/params into type safe functions
     // paths["/v0/solar/GB/gsp/forecast/all/"]["get"]["responses"]["200"]["content"]["application/json"]
     components["schemas"]["OneDatetimeManyForecastValues"][]
   >(
     // `/v0/solar/GB/gsp/forecast/all/`,
-    `${API_PREFIX}/solar/GB/gsp/forecast/all/?historic=true&compact=true`,
-    {}
+    `${API_PREFIX}/solar/GB/gsp/forecast/all/?historic=true&compact=true&start_datetime_utc=${encodeURIComponent(
+      `${forecastLastFetch30MinISO.slice(0, 19)}+00:00`
```

Comment:
this feels a bit iffy, not sure there is a different way to do it

Reply:
Yeah, not my favourite, but seems like the API expects the ISO date in different formats for different things I think?